modules/redis/redis.go

```diff
@@ -27,10 +32,25 @@ const (
 
 type RedisContainer struct {
 	testcontainers.Container
+	settings options
+}
+
+// Terminate terminates the Redis container, removing the TLS certificates that were created in the fly.
```

Comment:
suggestion: remove if the above suggestion to no set ParentDir is valid as it won't be need.